README.md

```diff
@@ -58,7 +58,7 @@ distribution, found at https://www.anaconda.com/download/).
 Once this is installed, the following command will install all required packages in your Python environment:
 
 ```
-conda env create -f environment.yml
```

Comment:
I thought about this a bit... We're teaching at a special conference, so maybe the user might be making a bunch of different conda environments and this makes sense. But, when we do one-offs, we might want to just dump it into root, cause we want to just get to a working environment. 

Also, if we do want a name, I think it should be put it in the `environment.yml` as

```
- name: scipygeo2018
```